pkg/controller/datavolume/conditions.go

```diff
@@ -134,11 +134,16 @@ func updateBoundCondition(conditions []cdiv1.DataVolumeCondition, pvc *corev1.Pe
 				conditions = UpdateReadyCondition(conditions, corev1.ConditionFalse, "", "")
 			}
 		case corev1.ClaimPending:
+			pvcPrimeName := getPrimeMessage(pvc)
+			if pvcPrimeName != "" {
+				// add space for readability
+				pvcPrimeName = " " + pvcPrimeName
+			}
 			if pvcCondition == nil || pvcCondition.Status == corev1.ConditionTrue {
-				conditions = updateCondition(conditions, cdiv1.DataVolumeBound, corev1.ConditionFalse, fmt.Sprintf("PVC %s Pending", pvc.Name), pvcPending)
+				conditions = updateCondition(conditions, cdiv1.DataVolumeBound, corev1.ConditionFalse, fmt.Sprintf("PVC %s Pending%s", pvc.Name, pvcPrimeName), pvcPending)
```

Comment:
Wouldn't it be cleaner to include the space inside the string "PVC %s Pending %s" instead of adding it separately? It feels less hacky imo.